### PR TITLE
Eventlist ticker

### PR DESCRIPTION
**Pull request by getnashty**
# Trello card link

# Checklist before requesting a review

- [ ] I have performed a self-review of my code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published

## Acceptance

https://www.notion.so/aiera/Test-Flows-Template-5203736ccfcb424c81643065f2bbf5e2

- [ ] I have reviewed the Test Flows template and it is up-to-date after considering my changes

### CODE REVIEW

File: src/modules/EventList/index.tsx [hunk 1]
```diff
@@ -71,15 +74,186 @@ export interface EventListUIProps {
...
+    index: number;
+    isRefetching: boolean;
+    onSelectEvent?: ChangeHandler<EventListEvent>;
+    renderedRefetch: boolean;
+    searchTerm?: string;
+    setFocus?: Dispatch<SetStateAction<number>>;
+    showDivider: boolean;
+}
+
+const EventRow = ({
```
Comment (getnashty) [inline src/modules/EventList/index.tsx:94]: using the EventRow twice... in upcoming events query, and then the recent events query

File: src/modules/EventList/index.tsx [hunk 2]
```diff
@@ -71,15 +74,190 @@ export interface EventListUIProps {
...
+    isRefetching,
+    onSelectEvent,
+    refetch,
+    renderedRefetch,
+    searchTerm,
+    index,
+    setFocus,
+    showDivider,
+}: EventRowProps) => {
+    const hitRatio = (numMentions || 0) / maxHits;
```
Comment (mmichael0413) [inline src/modules/EventList/index.tsx:107]: If `maxHits` is 0, you'll get a `NaN` here, so might be worth making this a ternary so when `maxHits` is 0, then `hitRatio` can be 0 too

File: src/modules/EventList/index.tsx [hunk 3]
```diff
@@ -71,15 +74,190 @@ export interface EventListUIProps {
...
+                                    </span>
+                                </>
+                            ) : (
+                                <span className="leading-none text-sm text-black dark:text-white truncate font-bold">
+                                    {event.title}
+                                </span>
+                            )}
+                        </div>
+                        <div className="leading-none flex text-sm capitalize items-center mt-1 text-black dark:text-white">
+                            {event.eventType.replace(/_/g, ' ')}
```
Comment (mmichael0413) [inline src/modules/EventList/index.tsx:215]: Do we want to titleize this or nah?

Comment (getnashty) [reply src/modules/EventList/index.tsx:215]: Nah

**Review by mmichael0413 — APPROVED** [review]

**Pull request merged**